doc/example/basic.txt

```diff
@@ -151,12 +151,7 @@ By default tox uses `pip`_ to install packages, both the
 package-under-test and any dependencies you specify in ``tox.ini``.
 You can fully customize tox's install-command through the
 testenv-specific :confval:`install_command=ARGV` setting.
-For instance, to use ``easy_install`` instead of `pip`_::
-
-    [testenv]
-    install_command = easy_install {opts} {packages}
-
```

Comment:
We should not simply delete that but use a different example. I wish we could use conda already, so I am a bit stumped for a good example. pip with different parameters would be something ... or something far out like:

    install_command = sudo apt install {opts} {packages}

which is obviously crazy but shows that you can stick everything in there, whether it makes sense or not.